### ISSUE TITLE
draggable cursor doesn't always work (in chrome)

### ISSUE THREAD

**Issue by boldwade** [akserg/ng2-dnd#171]
* **I'm submitting a ...**
[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**
Bug

* **What is the current behavior?**
Creating a draggable handle, sometimes the component doesn't drag. 

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via https://plnkr.co or similar.
Dynamically create elements with draggable handles, open in chrome and see that it will sometimes not allow the component to drag.
Seems to work fine in every other browser except Chrome.

* **What is the expected behavior?**
Should drag the handle every time.

* **What is the motivation / use case for changing the behavior?**
So that it works as expected.

* **Please tell us about your environment:**

- Angular version: 4.1.3
- Browser: Chrome Version 59.0.3071.115 (Official Build) (64-bit)

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by capi** [comment]
I do have the same issue as well. Creating a dynamic `<span dnd-draggable-handle>=</span>` works in Firefox, but not in Chrome.

**Comment by boldwade** [comment]
Yeah it works in FF, IE11, Edge & Safari... just not Chrome... which is the complete opposite of what I USUALLY say about browser issues. :)

**Comment by GuiBGP** [comment]
Hey, I think it´s duplicated (see #80).
